.changeset/real-numbers-bake.md

```diff
@@ -0,0 +1,8 @@
+---
+"@keystone-6/core": major
```

Comment:
Technically a breaking change? ... custom Navigation in the Admin UI will render weirdly or have invalid DOM structure if not updated.